requirements.txt

```diff
@@ -18,18 +18,17 @@ Jinja2==2.10.1; python_version < '3.8'
 # bullseye version, focal patched 2.10
 Jinja2==2.11.2; python_version >= '3.8'
 libsass==0.17.0
-lxml==3.7.1 ; sys_platform != 'win32' and python_version < '3.7'
-lxml==4.3.2 ; sys_platform != 'win32' and python_version == '3.7'
+lxml==4.3.2 ; sys_platform != 'win32' and python_version <= '3.7'
 lxml==4.6.1 ; sys_platform != 'win32' and python_version > '3.7'
 lxml ; sys_platform == 'win32'
 Mako==1.0.7
 MarkupSafe==1.1.0
 num2words==0.5.6
 ofxparse==0.19
 passlib==1.7.1
-Pillow==5.4.1 ; python_version <= '3.7' and sys_platform != 'win32'
+pikepdf==1.19.0
```

Comment:
I agree that we cannot use that until the Bullseye [release](https://wiki.debian.org/DebianReleases) as Buster ships only [pikepf 1.0.5](https://packages.debian.org/buster/python3-pikepdf).

Actually your PR fails on runbot because pip 9.0 shipped with bionic fails to find the  `wheel` package for pikepdf 1.19.0. We could use a Focal Docker image to test your  branch.

One thing that bother me is that pikepdf [mention](https://pikepdf.readthedocs.io/en/latest/release_notes.html#v2-0-0b1) that it only supports python 3.9 since pikepdf 2.0.0b1 ... So, the feature should be tested with a python 3.9 as it will be the default in Bullseye.

Reply:
I also noticed the coupling with the pip version when installing pikepdf 2.3.0 (the latest version): I needed at least pip 20.something to be able to use the wheel.

Nice catch on the 3.9. So I guess Debian is committed to fixing any compatibility issues they encounter between Python 3.9 and the 1.17.3 pikepdf they include in bullseye. It would further complicate our life if they find something and backport it from a newer version, because then we need to bump the version in requirements.txt again. There's a bit more than 2 months of development between 1.19.0 and 2.0.0b1.